### ISSUE TITLE
swag_from does not accept pathlib.Path objects as path specification

### ISSUE THREAD

**Issue by frankcorneliusmartin** [flasgger/flasgger#251]
I think it would be nice if the swag_from decorator would accept `pathlib.Path` objects as this makes the paths os independent. Thus instead of:
```
@swag_from("swagger/version.yaml", endpoint='version')
def get(self):
....
```
I rather use:
```
@swag_from(Path("swagger/version.yaml"), endpoint='version')
def get(self):
....
```
I now have to wrap `Path("swagger/version.yaml")` into `str()`

**Comment by lquastana** [comment]
Hello, I would like to contribute to your project, can I work on this issue ?

**Comment by billyrrr** [comment]
Yep, that would be really helpful. Thanks! Feel free to comment on this thread if you have any question at all.
